CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contributing to Prettier for PHP
+
+To get up and running, install the dependencies and run the tests:
+
+```bash
+yarn
+yarn test
+```
+
+Here's what you need to know about the tests:
+
+* The tests use [Jest](https://facebook.github.io/jest/) snapshots.
+* You can make changes and run `jest -u` (or `yarn test -u`) to update the snapshots. Then run `git diff` to take a look at what changed. Always update the snapshots when opening a PR.
+* You can run `AST_COMPARE=1 jest` for a more robust test run. That formats each file, re-parses it, and compares the new AST with the original one and makes sure they are semantically equivalent.
```

Comment:
maybe `AST_COMPARE=1 yarn test` for consistency?

Reply:
:+1: done